user/deployment/bluemixcloudfoundry.md

```diff
@@ -18,7 +14,7 @@ Go grab [the Travis gem from GitHub](https://github.com/travis-ci/travis.rb) and
 travis setup bluemixcloudfoundry
 ```
 
-You will need the following information about your Bluemix environment: username, password, organization, space, and region. Available Bluemix regions are US South (ng) London (eu-gb), and Sydney (au-syd). Travis offers to encrypt your passwod, and will take care of the rest. Learn more about [managing organizations and spaces](https://console.ng.bluemix.net/docs/admin/orgs_spaces.html).
+You will need the following information about your Bluemix environment: username, password, organization, space, and region. Available Bluemix regions are US South (ng) London (eu-gb), and Sydney (au-syd). Travis offers to encrypt your passwod, and will take care of the rest. Learn more about [managing organizations and spaces](http://bluemix.net/docs/admin/orgs_spaces.html).
```

Comment:
s/passwod/password/

I probably missed that last time. 

Otherwise looks fine.